modules/ROOT/pages/runtime-fabric-logs.adoc

```diff
@@ -165,7 +165,8 @@ Example output plugin configuration:
 | Trace Error | When enabled, prints the Elasticsearch API calls to stdout when Elasticsearch returns an error. | No | Off
 | Current Time Index | Specifies using the current time for index generation instead of message record information. | No | Off
 | Logstash Prefix Key | Prefixes keys with this string. | No |
-| Tls | Enables or disables TLS support. | No | Off
+| TLS | Enables or disables TLS support. | No | Off
+| CA Path Certificate File |  Specifies the CA certificate file to be uploaded. This option is available only when TLS is enabled. | No |
 |===
 
 Example output plugin configuration:
```

Comment:
For all of these, are we saying "here's an example configuration for an output plugin (what's an output plugin?) 
Or are we missing a preposition, like "Example output {that results from the plugin configuration above}? 
Or do we mean "Example values for the OUTPUT parameter of the (Azure log analytics plugin?)"?
I try to avoid noun phrases of more than three nouns.